Favor/Favor/Sources/Scenes/Gift/ViewControllers/NewGiftFriendVC.swift

```diff
@@ -80,53 +140,12 @@ final class NewGiftFriendViewController: BaseViewController, View {
     return view
   }()
   
-  private var headerView: NewGiftFriendHeaderView?
-  
-  // MARK: - Properties
-  
-  private lazy var dataSource: DataSource = DataSource(
-    configureCell: { _, collectionView, indexPath, item in
-      switch item {
-      case .empty:
-        let cell = collectionView.dequeueReusableCell(for: indexPath) as NewGiftFriendEmptyCell
-        return cell
-      case .friend(let reactor):
-        let cell = collectionView.dequeueReusableCell(for: indexPath) as NewGiftFriendCell
-        cell.reactor = reactor
-        return cell
-      }
-    },
-    configureSupplementaryView: { dataSource, collectionView, kind, indexPath in
-      let sectionModel = dataSource[indexPath.section]
-      
-      if kind == UICollectionView.elementKindSectionHeader {
-        // Header
-        let header = collectionView.dequeueReusableSupplementaryView(
-          ofKind: kind,
-          for: indexPath
-        ) as NewGiftFriendHeaderView
-        header.reactor = NewGiftFriendHeaderViewReactor(sectionModel: sectionModel)
-        header.textFieldChanged = { [weak self] in
-          self?.reactor?.action.onNext(.textFieldDidChange($0))
-        }
-        return header
-      } else {
-        // Footer
-        let footer = collectionView.dequeueReusableSupplementaryView(
-          ofKind: kind,
-          for: indexPath
-        ) as NewGiftFriendFooterView
-        return footer
-      }
-    }
-  )
-  
   // MARK: - Setup
   
   override func setupStyles() {
     super.setupStyles()
     
-    self.navigationItem.rightBarButtonItem = UIBarButtonItem(customView: self.doneButton)
+    self.navigationItem.rightBarButtonItem = UIBarButtonItem(customView: self.finishButton)
```

Comment:
```suggestion
    self.navigationItem.rightBarButtonItem = self.finishButton.toBarButtonItem()
```

이거 내가 `UIButton` `extension`으로 만들어뒀어!